packages/html-reporter/src/testCaseView.tsx

```diff
@@ -46,8 +46,13 @@ export const TestCaseView: React.FC<{
   }, [test]);
 
   const visibleAnnotations = React.useMemo(() => {
-    return test?.annotations?.filter(annotation => !annotation.type.startsWith('_')) || [];
-  }, [test?.annotations]);
+    if (!test)
+      return [];
+    const annotations = [...test.annotations];
+    if (test.results[selectedResultIndex])
+      annotations.push(...test.results[selectedResultIndex].annotations);
```

Comment:
@Skn0tt This will crash the reporter if I open up an older report that has no `annotations`, not sure if this is a realistic scenario?
```suggestion
      annotations.push(...test.results[selectedResultIndex]?.annotations ?? []);
``` 
![image](https://github.com/user-attachments/assets/53370e35-36c6-4eb1-a3e8-51bf7f8306c2)

Reply:
The old report will contain an old version of the javascript, so there's no need for cross-version considerations here.